cli/use.go

```diff
@@ -37,15 +37,21 @@ func (z *ZVM) Use(ver string) error {
 func (z *ZVM) setBin(ver string) error {
 	// .zvm/master
 	version_path := filepath.Join(z.baseDir, ver)
-	if err := os.Remove(filepath.Join(z.baseDir, "bin")); err != nil {
-		return err
+	bin_dir := filepath.Join(z.baseDir, "bin")
+
+	// Remove "bin" dir only if it already exists
+	_, err := os.Stat(bin_dir)
+	if !os.IsNotExist(err) {
```

Comment:
This is one of those unfortunate unclear areas of Go. The `os.IsNotExist/Exist` functions are legacy, and don't work for errors produced outside of the OS package. Best practices (to avoid unintended bugs) is as follows:

[Here's the API](https://pkg.go.dev/os#IsExist) that should be used.

![image](https://github.com/tristanisham/zvm/assets/23124818/582243aa-8e7e-442d-8e1e-69429e30a929)

Could you please update this bit to use the new API?

Reply:
Wow, ok I think my IDE proposed to use the `IsNotExist`, or maybe it was just me doing not enough research 😅 thanks for the explanation. And I saw you already adjusted the code, thanks 🙏🏼